apps/ensapi/src/index.ts

```diff
@@ -45,9 +47,29 @@ app.route("/api", ensNodeApi);
 // use Subgraph GraphQL API at /subgraph
 app.route("/subgraph", subgraphApi);
 
+// use ENSAnalytics API at /ensanalytics
+app.route("/ensanalytics", ensanalyticsApi);
+
 // will automatically 500 if config is not available due to ensIndexerPublicConfigMiddleware
 app.get("/health", async (c) => {
-  return c.json({ ok: true });
+  const cache = cacheService.getCache();
+  const ensanalyticsOk = cacheService.isCacheReady();
+
+  // overall health is ok only if all subservices are healthy
+  const allOk = ensanalyticsOk;
+
+  return c.json({
+    ok: allOk,
+    ensanalytics: {
+      ok: ensanalyticsOk,
+      cache: cache
+        ? {
+            totalReferrers: cache.referrers.length,
+            updatedAt: cache.updatedAt,
+          }
+        : null,
+    },
+  });
```

Comment:
@lightwalker-eth Should we just leave it as it was, with a single "ok" field, but affected by the cache status? Or is adding this separate field for the ensanalytics health ok?